master/getting-started/kubernetes/troubleshooting.md

```diff
@@ -36,3 +36,17 @@ depend on how your kubelet is configured.  For deployments using `systemd`, you
 
 The log level can be configured via the CNI network configuration file, by changing the value of the
 key `log_level`.  See [the configuration guide]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration) for more information.
+
+#### How do I specify the IP Pools to use to assign pod IPs? 
```

Comment:
I think the question we're answering in terms of the a Kubernetes user's expectations is actually "How do I configure the Pod IP range?"

The answer is "When using Calico IPAM, IP addresses are assigned from [IP Pools](link). . ."